src/slo/backup_quality/quality_sli_service.py

```diff
@@ -28,19 +31,31 @@ def recalculate_sli(self):
         logging.info("Recalculating quality SLI has been started.")
 
         all_tables, snapshot_time = self.querier.query()
-        filtered_tables = [table for table in all_tables
-                           if self.__should_stay_as_sli_violation(table)]
+        tasks = []
+        for table in all_tables:
+            tasks.append(
+                Task(
+                    method='POST',
+                    url='/sli/quality/violation',
+                    payload=json.dumps({'table': table, 'snapshot_time': snapshot_time}),
+                    headers={'Content-Type': 'application/json'}
+                )
+            )
+        if tasks:
+            Tasks.schedule('sli-table-quality-violations', tasks)
 
-        logging.info("Quality SLI tables filtered from %s to %s", len(all_tables), len(filtered_tables))
-        self.streamer.stream(
-            filtered_tables,
-            snapshot_marker=self.__create_snapshot_marker_row(snapshot_time)
-        )
+        self.streamer.stream([], snapshot_marker=self.__create_snapshot_marker_row(snapshot_time))
```

Comment:
We should change somehow streamer method. It doesn't need an empty array. Same with SliLatencyService.

Reply:
done